ygen/gogen.go

```diff
@@ -637,6 +655,20 @@ func (t *{{ .Receiver }}) GetOrCreate{{ .ListName }}(
 	}
 	return v
 }
+`
+
+	// goLeafGetterTemplate defines a template for a function that, for a
+	// particular leaf, generates a getter method.
+	goLeafGetterTemplate = `
+// Get{{ .Name }} retrieves the value of the leaf {{ .Name }} from the {{ .Receiver }}
+// struct. Caution should be exercised whilst using this method since it will return
+// the Go zero value if the field is explicitly unset.
```

Comment:
(optional) Do we need to worry about getting out of sync with the "Go zero values" since we aren't consulting Go when setting them, instead we are creating a maps of zero values maintained within this package.

Go shouldn't really change this because of their compatibility guarantee, just something to think about.